src/experiment/launch_experiment.jl

```diff
@@ -63,16 +64,18 @@ function launch_experiment!(
 
         for j in 1:nbHeuristics
             reset_model!(model)
-            
-            dt = @elapsed search!(model, strategy, variableHeuristic, valueSelectionArray[j], out_solver=out_solver)
-            
-            metricsArray[j](model,dt)  #adding results in the metrics data structure
+            dt = @elapsed for k in 1:restartPerInstances
+                restart_search!(model)
```

Comment:
reset some statistics on the CPModel for the search ! 
IMPORTANT, it doesn't reset the `objectiveBound` value. Hence, at each restart, the solution (if one is found) will be better.